src/api/controllers/popularArticleController.js

```diff
@@ -0,0 +1,73 @@
+
+import async from 'async';
+import models, { sequelize } from '../models';
+import status from '../../helpers/constants/status.codes';
+
+
+const {
+  Read, Article, User, Category
+} = models;
+
+/**
+ * containing all user's
+ * profile model controller(update, getUserProfile, read)
+ * @export
+ * @class ProfileController
+ */
+export default class PopularController {
+  /**
+         *
+         * get 5 mostly read article in the database
+         * @static
+         * @param {*} req The request Object
+         * @param {*} res The response Object
+         * @memberof PopularController
+         * @return {Object} The response Object
+         */
+  static async getAll(req, res) {
+    let resultArray = [];
+    const result = await Read.findAll({
+      group: ['articleSlug'],
+      attributes: ['articleSlug', [sequelize.fn('COUNT', 'articleSlug'), 'count']],
+      order: [
+        ['count', 'DESC']
+      ],
+      limit: 5
+    });
+
+    const slugs = result.map(item => item.dataValues.articleSlug);
```

Comment:
@MCFrank16 instead of multiple queries like this, since the Read model is associated with both the User and Article models, why not include those models in the query above? 

Since Category is associated with Article, you can include Category as well via nested includes: https://sequelize.readthedocs.io/en/latest/docs/models-usage/index.html#nested-eager-loading

Let me know what you think.